packages/web-sdk/src/config/makeCoreConfig.ts

```diff
@@ -91,6 +91,7 @@ export function makeCoreConfig(browserConfig: BrowserConfig): Config | undefined
     view: browserConfig.view,
     trackResources: browserConfig.trackResources,
     trackWebVitalsAttribution: browserConfig.trackWebVitalsAttribution,
+    consoleInstrumentation: browserConfig.consoleInstrumentation,
```

Comment:
Should we add some documentation now that the console instrumentation is being added by default or was this the default behaviour before?

Reply:
I already updated the cloud docs. 
Link is in the PR description. 
Anything missing there?